### ISSUE TITLE
RA2 mod crashed when libya exploding truck exploded near castle building

### ISSUE THREAD

**Issue by sorcerer86pt** [OpenRA/ra2#553]
Was playing Alamo map  with libya when i deployed the exploding truck near the base when the game crashed with the following:

```
OpenRA engine version release-20180923
Red Alert 2 mod version release-20180923
on map 982d0a37b50154eab6c3652b37301bb7966b6833 (The Alamo by Westwood Studios).
Date: 2019-04-14 13:05:06Z
Operating System: Windows (Microsoft Windows NT 6.2.9200.0)
Runtime Version: .NET CLR 4.0.30319.42000
Exception of type `System.InvalidOperationException`: Unit `castl01` does not have a sequence named `rubble`
   em OpenRA.Graphics.SequenceProvider.GetSequence(String unitName, String sequenceName) em g:\ra2\engine\OpenRA.Game\Graphics\SequenceProvider.cs:line 78
   em OpenRA.Graphics.Animation.GetSequence(String sequenceName) em g:\ra2\engine\OpenRA.Game\Graphics\Animation.cs:line 218
   em OpenRA.Graphics.Animation.PlaySequence(String sequenceName) em g:\ra2\engine\OpenRA.Game\Graphics\Animation.cs:line 99
   em OpenRA.Graphics.Animation.PlayRepeating(String sequenceName) em g:\ra2\engine\OpenRA.Game\Graphics\Animation.cs:line 107
   em OpenRA.Mods.Common.Traits.Render.WithSpriteBody..ctor(ActorInitializer init, WithSpriteBodyInfo info, Func`1 baseFacing) em g:\ra2\engine\OpenRA.Mods.Common\Traits\Render\WithSpriteBody.cs:line 69
   em OpenRA.Mods.Common.Traits.Render.WithSpriteBody..ctor(ActorInitializer init, WithSpriteBodyInfo info) em g:\ra2\engine\OpenRA.Mods.Common\Traits\Render\WithSpriteBody.cs:line 53
   em OpenRA.Mods.Common.Traits.Render.WithSpriteBodyInfo.Create(ActorInitializer init) em g:\ra2\engine\OpenRA.Mods.Common\Traits\Render\WithSpriteBody.cs:line 33
   em OpenRA.Actor..ctor(World world, String name, TypeDictionary initDict) em g:\ra2\engine\OpenRA.Game\Actor.cs:line 102
   em OpenRA.World.CreateActor(Boolean addToWorld, String name, TypeDictionary initDict) em g:\ra2\engine\OpenRA.Game\World.cs:line 266
   em OpenRA.World.CreateActor(String name, TypeDictionary initDict) em g:\ra2\engine\OpenRA.Game\World.cs:line 261
   em OpenRA.Mods.Common.Traits.SpawnActorOnDeath.<>c__DisplayClass8.<OpenRA.Traits.INotifyRemovedFromWorld.RemovedFromWorld>b__5(World w) em g:\ra2\engine\OpenRA.Mods.Common\Traits\SpawnActorOnDeath.cs:line 145
   em OpenRA.World.Tick() em g:\ra2\engine\OpenRA.Game\World.cs:line 373
   em OpenRA.Game.InnerLogicTick(OrderManager orderManager) em g:\ra2\engine\OpenRA.Game\Game.cs:line 616
   em OpenRA.Game.LogicTick() em g:\ra2\engine\OpenRA.Game\Game.cs:line 640
   em OpenRA.Game.Loop() em g:\ra2\engine\OpenRA.Game\Game.cs:line 770
   em OpenRA.Game.Run() em g:\ra2\engine\OpenRA.Game\Game.cs:line 810
   em OpenRA.Game.InitializeAndRun(String[] args) em g:\ra2\engine\OpenRA.Game\Game.cs:line 256
   em OpenRA.Program.Main(String[] args) em g:\ra2\engine\OpenRA.Game\Support\Program.cs:line 37
```

**Comment by MustaphaTR** [comment]
CASTL01 seems to be this building, and i can not reproduce the crash when i kill the building.
![resim](https://user-images.githubusercontent.com/7933210/56094389-8b09c480-5edc-11e9-9a41-34ea2c9c4e3a.png)

**Comment by abcdefg30** [comment]
Thanks for the report! I filed a fix at #554.

**Comment by abcdefg30** [comment]
@MustaphaTR is your version using the shortened inheritance already?
